draft-ietf-rats-corim.md

```diff
@@ -1868,18 +1879,51 @@ If the `selection` criteria is not satisfied, then evaluation procedes to the ne
 |           | `authority`     | Optional    |
 |           | `cmtype`        | n/a         |
 |           | `profile`       | n/a         |
+|           | `domain-id`     | n/a         |
 | selection | `environment`   | Mandatory   |
 |           | `element-list`  | Mandatory   |
 |           | `authority`     | Optional    |
 |           | `cmtype`        | n/a         |
 |           | `profile`       | n/a         |
+|           | `domain-id`     | n/a         |
 | addition  | `environment`   | Mandatory   |
 |           | `element-list`  | Mandatory   |
 |           | `authority`     | Mandatory   |
 |           | `cmtype`        | Mandatory   |
 |           | `profile`       | Optional    |
+|           | `domain-id`     | n/a         |
 {: #tbl-ev-ect-optionality title="Endorsed Values and Endorsed Values Series tuples requirements"}
 
+### Internal Representation of Domain Membership {#sec-ir-dm}
+
+An internal representation of Domain Membership uses the `dm` relation, which is a list of ECTs that describes the composition of an Attester with an identity of the Domain specified in the `identity` ECT.
+
+~~~ cddl
+{::include cddl/intrep-domain-mem.cddl}
+~~~
+
+The Domain member Environments are compared with the Environments specified in the list of ECTs in ACS whose `cmtype` matches Evidence.
```

Comment:
```suggestion
The Domain member Environments are compared with the Environments specified in the list of ECTs in ACS whose `cmtype` matches `&(evidence: 2)`.
```

Reply:
This sentence is quite confusing. What are the environment comparisons supposed to do?

Reply:
Removed it  from here and moved it to Processing of DMT section